content/projects/java-specific/functional-interface/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - topic/oop
+  - topics/java-specific/functional-interface
+ready: true
+submission_type: repo
+tags:
+- functional-interface
+- lambda-expressions
+title: Java Lamba Expressions and Functional Interfaces
+---
+
+### What is the number
```

Comment:
The html that comes out of this will be:

```
<h1>Java Lamba Expressions and Functional Interfaces</h1>
<h3>What is the number</h3>
```

Can you try to keep the headings in order? h1, then h2 etc? You are probably optimizing for looks right now but looks will change, it will be nice to not have to go back and fix everything.